Web/www/index.html

```diff
@@ -0,0 +1,194 @@
+					<div id="pid_card" class="card_body text--medium">
+						<!--<br>
+						<p class="text--bold">Pitch : </p>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">P</label>
+						<input type="text" class="card_input">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">I</label>
+						<input type="text">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">D</label>
+						<input type="text">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+
+						<br><br>
+						<p class="text--bold">Roll : </p>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">P</label>
+						<input id="roll_p" type="text" class="card_input">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">I</label>
+						<input type="text">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">D</label>
+						<input type="text">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+
+						<br><br>
+						<p class="text--bold">Yaw : </p>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">P</label>
+						<input type="text" class="card_input">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">I</label>
+						<input type="text">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+						<br>
+						<label style="display:inline-block; width:10%;" class="text--medium">D</label>
+						<input type="text">
+						<button class="button_up text--bold" type="increment">∧</button>
+						<button class="button_down text--bold" type="increment">∨</button>
+
+						<br><br>
+						<div style="text-align: center;">
+							<button class="text--bold" type="submit">Submit</button>
+						</div> -->
+					</div>
+				</div>
+			</section>
+		</main>
+		<script>
+			var pid_card = document.getElementById("pid_card");
```

Comment:
```suggestion
			const pid_card = document.getElementById("pid_card");
```

Nowadays they have a version of Javascript called ES6. It has a whole bunch of nice improvements on Javascript of old.

One improvement that they made was better variable scoping. You can now declare variables as `const` or `let` which are much better variable definitions. [Here's](https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75#:~:text=%60const%60%20is%20a%20signal%20that,always%20the%20entire%20containing%20function.) a crappy medium article that explains the difference.